integration_test.go

```
@@ -571,28 +569,30 @@ func TestNewConnectWithLowTimeout(t *testing.T) {
 						})
 					}
 
-					if tcase.controlQuery != DontRun {
-						t.Run("Query from control connection", func(t *testing.T) {
-							err = s.control.querySystem("SELECT key FROM system.local WHERE key='local'").err
-							match(t, tcase.controlQuery, err)
-						})
-					}
+				if tcase.controlQuery != DontRun {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)